src/modules/common/components/__tests__/__snapshots__/ScrollingSearchBox.spec.js.snap

```diff
@@ -34,7 +35,8 @@ exports[`ScrollingSearchBox should set correct reference 1`] = `
         />
         <input
           class="SearchInputstyles__TextInput-kJgKBC lfoCjL"
-          placeholder="common:search"
+          label="Placeholder"
```

Comment:
wrong snapshot, I think you just need to update them.

Reply:
I was sure I did update them o.O